modules/tinymce/src/plugins/table/main/ts/core/Util.ts

```diff
@@ -37,14 +37,21 @@ const addSizeSuffix = function (size: string) {
   return size;
 };
 
-const removeDataStyle = (table) => {
+const removeDataStyle = (table: Element<HTMLElement>) => {
   const dataStyleCells = SelectorFilter.descendants(table, 'td[data-mce-style],th[data-mce-style]');
   Attr.remove(table, 'data-mce-style');
   Arr.each(dataStyleCells, function (cell) {
     Attr.remove(cell, 'data-mce-style');
   });
 };
 
+const getRawWidth = (editor: Editor, elm: HTMLElement) => {
```

Comment:
This function looks familiar. There's another one like it somewhere else in here. Why are they different?

Is this function and its friend adequately tested?

Can we add return types to these functions?

Perhaps we should have a `Strings.isEmpty` and `Strings.isNotEmpty` in katamari.

Reply:
Added new `isEmpty` and `isNotEmpty` functions to `Strings` and added return types. This bit of code runs on many tests (eg the new `DefaultTableSizingTest`), so yes to adequately tested (though not unit tested).